.github/workflows/ci.yml

```diff
@@ -205,7 +205,7 @@ jobs:
         if: ${{ matrix.docs }}
         run: |
           cd docs/sphinx-docs/
-          python build_sphinx.py || true
+          python build_sphinx.py
```

Comment:
Yes please to this... but I'm not sure that it is routinely passing as a matter of course at this stage.

Reply:
we'll see what it does, but no, its not passing. Which is making figuring out the problem with the other changes harder.

Reply:
I suspect that all the `check_call`uses need to be tweaked so that if it fails, stdout/stderr end up in the log as part of the complaint, rather than disappearing.

Reply:
(yes, I know it's not *currently* passing, but I'm not sure it actually does on a good day... it's currently failing at apidoc but I think it actually normally returns errors from sphinx itself due to sketchy crossrefs etc)

Reply:
If apidoc is now hitting Qt code and used not to, it might be that it has to be run inside xvfb-run so that there's a display so that pyqt stuff can be imported.

Reply:
I'm not sure that its hitting very much new in this branch, the omission stuff was all out of date. But maybe it has been for a while.